QuickReference.ipynb

```diff
@@ -0,0 +1,402 @@
+    "| **waveband** | Resulting services have data in the specified waveband(s) | ‘radio’, ‘millimeter’, ‘infrared’, ‘optical’, ‘uv’, ‘euv’, ‘x-ray’ ‘gamma-ray’ |\n",
+    "\n",
+    "### 2.1.2 Inspect the results.\n",
+    "#### Using pyvo\n",
+    "Although not lists, `pyvo` results can be iterated over to see each individual result.  The results are specialized based on the type of query, providing access to the important properties of the results.  Some useful accessors with registry results are:\n",
+    "* `short_name` - A short name\n",
+    "* `res_title` - A more descriptive title\n",
+    "* `res_description` - A more verbose description\n",
+    "* `reference_url` - A link for more information\n",
+    "* `ivoid` - A unique identifier for the service.  Gives some indication of what organization is serving the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print the number of results and the 1st 4 short names and titles.\n",
+    "print(f'Number of results: {len(services)}\\n')\n",
+    "for s in list(services)[:4]:  # (Treat services as list to get the subset of rows)\n",
+    "    print(f'{s.short_name} - {s.res_title}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Filtering results\n",
+    "Of the services we found, which one(s) have 'stsci.edu' in their unique identifier?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stsci_services = [s for s in services if 'stsci.edu' in s.ivoid]\n",
+    "for s in stsci_services:\n",
+    "    print (f'(STScI): {s.short_name} - {s.res_title}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Using astropy\n",
+    "With the `to_table()` method, `pyvo` results can also be converted to Astropy `Table` objects which offer a variety of addional features. See http://docs.astropy.org/en/stable/table/ for more on working with Astropy Tables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Convert to an Astropy Table\n",
+    "services_table = services.to_table()\n",
+    "\n",
+    "# Print the column names and display 1st 3 rows with a subset of columns\n",
+    "print(f'\\nColumn Names:\\n{services_table.colnames}\\n')  \n",
+    "services_table['short_name', 'res_title', 'res_description'][:3]  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.2 Cone search\n",
+    "Example:  Find a cone search service for the USNO-B catalog and search it around M51 with a .1 degree radius.  (More inspection could be done on the service list instead of blindly choosing the first service.)  \n",
```

Comment:
It's not clear what the "More inspection could be done" parenthetical is talking about until one has read the code, suggest moving to its own sentence at the end of the cell, that says something like "In this example, we simply choose the first service returned, however more inspection could be done to determine the best service to choose."